api/integreatly/v1alpha1/grafanadashboard_types.go

```diff
@@ -81,7 +81,7 @@ type GrafanaDashboardRef struct {
 }
 
 type GrafanaDashboardStatus struct {
-	Content          string                 `json:"content,omitempty"`
+	ContentGzip      []byte                 `json:"contentGzip,omitempty"`
```

Comment:
Since you use the name ContentGzip does this only now provide content that is related if you are using the GzipJson?
If not I guess something like ContentCompressed could be a better name to  make sure that users don't draw that conclusion.

Reply:
Renamed to contentCache which should be pretty clear I think.